iocore/net/SSLCertLookup.cc

```diff
@@ -369,7 +370,7 @@ reverse_dns_name(const char *hostname, char (&reversed)[TS_MAX_HOST_NAME_LEN + 1
 }
 
 SSLContextStorage::SSLContextStorage()
-  : wilddomains(ink_hash_table_create(InkHashTableKeyType_String)), hostnames(ink_hash_table_create(InkHashTableKeyType_String))
+  : wilddomains(HashMap<cchar *, StringHashFns, int>(-1)), hostnames(ink_hash_table_create(InkHashTableKeyType_String))
```

Comment:
Why is this initialized at all? And if it needs initialization, why not just pass the `-1` directly instead of constructing a temporary?